modules/menu/menu.c

```diff
@@ -379,6 +379,10 @@ static void ua_event_handler(struct ua *ua, enum ua_event ev,
 #endif
 
 
+	if (call_state(call) == CALL_STATE_TERMINATED &&
+			ev != UA_EVENT_CALL_CLOSED)
+		return;
+
```

Comment:
this looks like a hack. there are also events that are not call related

Reply:
Added a commit. Check if call is set, doesn't hurt and maybe makes it clearer.

Reply:
I disagree with this. The event handler should handle the event, even if the call state is closed.

The `struct call` code has to robust enough to check its own state